govcd/task.go

```diff
@@ -126,6 +128,22 @@ func (task *Task) WaitInspectTaskCompletion(inspectionFunc InspectionFunc, delay
 			return nil
 		}
 
+		// If the environment variable "GOVCD_TASK_MONITOR" is set, its value
```

Comment:
It seems we're missing a section in `TESTING.md` describing what environment variables there are. It's kind of a mystery right now.

Reply:
done

Reply:
Beautiful, thanks!